Tests/AppKernel.php

```diff
@@ -37,6 +38,15 @@ public function registerBundles() {
 
 	public function registerContainerConfiguration(LoaderInterface $loader) {
 		$loader->load($this->config);
+
+		if (class_exists('Symfony\Component\Asset\Package')) {
+			// enable assets to avoid fatal error "Call to a member function needsEnvironment() on a non-object in vendor/twig/twig/lib/Twig/Node/Expression/Function.php on line 25" with Symfony 3.0
```

Comment:
do you really need it ? Do you depend on AsseticBundle for your integration tests ?